### ISSUE TITLE
Dutch-Belgium resx file breaks ResxManager

### ISSUE THREAD

**Issue by Sako73** [tom-englert/ResXResourceManager#83]
I see that **nl-BE** is in the languages list, but when I try to add it as a language, it does not appear as an option in the dropdown.

When I have a *.nl-BE.resx file in the project, it only reads this file, and doesn't recognize others. When I remove that file, it works as expected.

Thanks.

**Comment by tom-englert** [comment]
It does no longer appear in the drop down if you already have a column of that language.

It works for me:
![image](https://user-images.githubusercontent.com/10229954/36253268-4fb52880-1247-11e8-96e5-19fc60c6abe7.png)

Can you post a sample where it does not work?

**Comment by Sako73** [comment]
[ECommerce.nl-BE.resx.zip](https://github.com/tom-englert/ResXResourceManager/files/1727820/ECommerce.nl-BE.resx.zip)
Yep, I saw that the language was missing because I already had the nl-BE added to the grid.

The attached file still breaks though. It will read this file, but won't recognize any others in the directory. If I remove this file, it parses the rest great.

Thanks.

**Comment by tom-englert** [comment]
I don't have problems with this, everything works fine...

**Comment by Sako73** [comment]
That file reads, but if it is in the directory, no other files, including the neutral language, will.

I'll look through the code some more. Thanks for looking into it

**Comment by tom-englert** [comment]
I have tried by cloning your sample file as neutral and German, and it all worked fine. Maybe there are other constraints? Can you reproduce this in a sample project?

**Comment by tom-englert** [comment]
@Sako73 you still have this problem?

**Comment by tom-englert** [comment]
Closed - not reproducible, and no feedback from @Sako73

**Comment by newyorker1984** [comment]
@tom-englert 

[resx files.zip](https://github.com/tom-englert/ResXResourceManager/files/2646235/resx.files.zip)

We are still having a problem with the nl-BE resx file.   Any help would be greatly appreciated.  The attached resx files are from our Visual Studio 2017 project.  

ResxManager (both desktop & VS plugin) will only load the nl-be file and not all the files.  remove the nl-be file and ResxManager will load all the files as expected.  What is odd is that the nl-BE & nl-NL files are identical.

We found that if we copy & rename the nl-NL file to nl-BE ResxManager will load all the files.  After a value edit to the nl-BE file, using visual studio, ResxManager (both desktop & VS plugin) will only load the nl-be file.

All files, only nl-BE loads
![image](https://user-images.githubusercontent.com/5477859/49474861-c1735c80-f7e3-11e8-9b82-ad853b97ef9b.png)

Removed nl-be file, all files load
![image](https://user-images.githubusercontent.com/5477859/49474899-da7c0d80-f7e3-11e8-98a2-b75b641d5862.png)
